### PR TITLE
Feature/add bukka logo to head

### PR DESCRIPTION
**Pull request by jimioniay**
#### What does this PR do?
Add logo to header, add title to header, add new "New No Nearby bukka" page

#### Description of the task to be completed?
- point 1
- point 2
- point 3

#### How can this manually be tested?
- clone this repo
```
$ git clone https://github.com/bukkaApp/Mybukka-Frontend
```
- install all dependencies
```
$ npm i
```
- start Application
```
$ npm start:dev
```
- log on to _localhost:PORT_ to interact with created page

#### Relevant PT story(ies)
<PT_STORY_ID>

#### Questions

### CODE REVIEW

File: src/features/feed/components/FreshSection.jsx [hunk 1]
```diff
@@ -66,11 +66,12 @@ const FreshSection = ({
   //   setCategories([...searchResult]);
   // }, [searches]);
 
-  if (bukkaMenu.length === 1 && error) {
+  // if (bukkaMenu.length === 1 && error) {
```
Comment (akolliy1) [inline src/features/feed/components/FreshSection.jsx]: Why do you comment out this line? Please add it back. It should only show when there is an error.

File: src/features/home/index.jsx [hunk 2]
```diff
@@ -11,9 +11,11 @@ import ChooseAreaToExploreSection from './components/ChooseAreaToExploreSection'
 
 import ReadyToOrderSection from './components/ReadyToOrderSection';
 
+
```
Comment (akolliy1) [inline src/features/home/index.jsx]: too much space

File: src/features/home/index.jsx [hunk 3]
```diff
@@ -11,9 +11,11 @@ import ChooseAreaToExploreSection from './components/ChooseAreaToExploreSection'
...
 const Home = ({ history: { push }, coordinates }) => {
   useEffect(() => () => push('/feed'), [coordinates]);
 
+  // coordinates.length > 0 && fetchNearByBukkas(coordinates);
```
Comment (akolliy1) [inline src/features/home/index.jsx]: remove this

**Review by akolliy1 — CHANGES_REQUESTED** [review]
Please review

File: src/features/feed/components/FreshSection.jsx (hunk 1 above)
Comment (jimioniay) [reply src/features/feed/components/FreshSection.jsx]: resolved

File: src/features/home/index.jsx (hunk 3 above)
Comment (jimioniay) [reply src/features/home/index.jsx]: resolved

File: src/features/home/index.jsx (hunk 2 above)
Comment (jimioniay) [reply src/features/home/index.jsx]: resolved

**Pull request merged**